app.js

```diff
@@ -39,5 +39,12 @@ app.use(bodyParser.json());
 app.use(express.static('browser/dist'));
 
 app.use('/', router);
-
+app.use((err, req, res, next) => {
+  res.status(500);
+  res.json({
+    errors: {
+      message: 'There was a server error. Please contact the site administrator.',
+    },
+  });
+});
```

Comment:
This is the express error handling middleware. All errors that have not been explicitly written and/or sent in our routes will get passed to this function and a 500 server error will get sent. This can be customized to send all errors if we want. We also can set up a version for d`DEV` that provides the stack trace. Right now the there is no stack trace. Let me know if you'd like me to add that.

Reply:
Custom error handling gets defined last, looks good.  We haven't been sending the error objects with other custom errors, so this is probably preferred.